src/shell/shell.cpp

```diff
@@ -1179,13 +1197,25 @@ void SHELL_Init() {
 	        "ls: cannot access '%s': No such file or directory\n");
 
 	MSG_Add("SHELL_CMD_CHOICE_HELP",
-	        "Waits for a keypress and sets ERRORLEVEL.\n");
+	        "Waits for a keypress and sets an ERRORLEVEL value.\n");
 	MSG_Add("SHELL_CMD_CHOICE_HELP_LONG",
-	        "CHOICE [/C:choices] [/N] [/S] text\n"
-	        "  /C[:]choices  -  Specifies allowable keys.  Default is: yn.\n"
-	        "  /N  -  Do not display the choices at end of prompt.\n"
-	        "  /S  -  Enables case-sensitive choices to be selected.\n"
-	        "  text  -  The text to display as a prompt.\n");
+	        "Usage:\n"
+	        "  \033[32;1mchoice\033[0m \033[36;1m[TEXT]\033[0m\n"
+	        "  \033[32;1mchoice\033[0m /c[:]\033[37;1mCHOICES\033[0m [/n] [/s] \033[36;1m[TEXT]\033[0m\n"
+	        "\n"
+	        "Where:\n"
+	        "  \033[36;1mTEXT\033[0m    is the text to display as a prompt, or empty.\n"
+	        "  \033[37;1mCHOICES\033[0m specifies allowable keys (default to \033[37;1myn\033[0m).\n"
+	        "\n"
```

Comment:
My comments here are similar to the `DIR` PR.

I think it's easier to read if the flags are listed, one by one, in the `Where:` block.

Something like this (feel free to adjust):

``` text
Where:
   /c[:]choices   Specifies allowable keys.  Default is: yn.
   /n             Do not display the choices at end of prompt.
   /s             Enables case-sensitive choices to be selected.
   TEXT           Is the text to display as a prompt, or empty.
```

And then the `Notes:` block can be any tips or suggestions for the flags that are already described in the `where:` block.  

So we're not introducing `/flag` for the first time embedded in a large paragraph of text.

Reply:
I initially missed the CHOICE command when I cleaned up help messages for DIR command, but found it after that. The options for CHOICE and DIR command may not be fully comparable though, since for DIR command I had to list the two options per line so that the total line count does not exceed 25, but for CHOICE command I guess it is okay to list one option per line since the total line count is under 25. I have moved option list for both commands under the "Where" block as you suggested.

Reply:
This reads very nice, thanks @Wengier !